Sources/Models/API/Taxonomies/Label.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Label.swift
+//  OpenFoodFacts
+//
+//  Created by Alexander Scott Beaty on 8/23/20.
+//
+
+import Foundation
+import RealmSwift
+import ObjectMapper
+
+class Label: Object {
+
+    @objc dynamic var code = ""
+
+    let parents = List<String>()
+    let children = List<String>()
+    let names = List<Tag>()
```

Comment:
After making the changes in my second commit (...Allow TaxonomyRequests to succeed when creating URLs..." 210e186), the data for labels was successfully saved to Realm. However, I just copied this model from Categories, and I noticed that almost all Labels and Categories have 'parents,' 'childre,' and 'names' as attributes, some of them have other attrbs. like 'country,' 'eu-groups,' 'auth-url,' to name a few. Not sure how to handle all of them or if this is enough.

Reply:
We probably don't use any of the attributes, in any of the taxonomies on iOS right now. If it's not too much extra work storing them, that would be useful, otherwise don't bother.

Reply:
Ok. The problem is, I don't know which attributes are common to all Labels, other than `parents, children, names.` I was trying to reverse-engineer the data model by picking random Labels in the database and looking at their attributes.

Reply:
It's constantly evolving and attributes are not present on all labels.